src/components/util/RivetIcons.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
+
+export const svg16 = (className: string = "", node: React.ReactNode) =>
+    <svg role="img" xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16" className={className}>
+        {node}
+    </svg>
+
+/****************
```

Comment:
Instead of these block comments, would you consider using

```javascript
//#region My region name
... some code
//#endregion
```

Since most code editors (including VSCode and IntelliJ) recognize that and then support code folding, which we wouldn't get with block comments.

Reply:
Will change, thanks for the suggestion.